pegasus/sites.v3/code.org/views/stats_homepage.haml

```diff
@@ -0,0 +1,21 @@
+- student_accounts_percent = "20%"
+- female_students = "9M"
+- teacher_users = "600K"
+- world_students_hoc_percent = "10%"
+- states_policy_change = "20"
+
```

Comment:
I have 2 questions about the way I've done this. 
1.) I assume at some point we want to translate the statistics and show for all languages... In which case, is it a problem for the number to be split from the phrase like I did for styling purposes? Will this lead to inaccurate translation?
2.) Will this info eventually be pulled from a database? If so, I put the info in variables, but I don't know the format of the numbers (e.g. will the % sign be included etc.)